deluge/_libtorrent.py

```diff
@@ -26,7 +26,7 @@
         raise LibtorrentImportError('No libtorrent library found: %s' % (ex))
 
 
-REQUIRED_VERSION = '1.1.2.0'
+REQUIRED_VERSION = '1.2.0.0'
```

Comment:
If we bump this version we can also remove any conditionals related to 1.2 elsewhere in code

Reply:
@cas-- I suggest that if you bump the requirement to 1.2.0. then you should go for at least 1.2.14 as there's been many bug and crash fixes in the later builds.